src/humanloop/client.py

```diff
@@ -78,11 +90,124 @@ def __init__(
             follow_redirects=follow_redirects,
             httpx_client=httpx_client,
         )
+
+        self._tracer_provider = TracerProvider(
```

Comment:
Want to see a proper docstring in this extended client and some inline comments explaining the OT integration.

Reply:
Ack